client/session_fetch_high_concurrency_test.go

```diff
@@ -201,3 +202,11 @@ func TestSessionFetchAllHighConcurrency(t *testing.T) {
 
 	require.NoError(t, session.Close())
 }
+
+func stringIDsToTsIDsIter(ids []string) ts.IDsIterator {
+	tsIDs := make([]ts.ID, 0, len(ids))
+	for _, id := range ids {
+		tsIDs = append(tsIDs, ts.StringID(id))
+	}
+	return ts.NewIDSliceIter(tsIDs)
```

Comment:
Hm, for convenience we should probably just create a copy of this method in the `ts` package.  i.e. `func NewStringsIDSliceIter(ids []string)`

Reply:
Yep will do